src/strings.cpp

```diff
@@ -1974,6 +1975,7 @@ bool ReadLanguagePack(const LanguageMetadata *lang)
 	if (U_FAILURE(status)) {
 		_current_collator.reset();
 	}
+	ICUParagraphLayoutFactory::InitializeLayouter();
```

Comment:
This is missing HARFBUZZ ifdef-guard

Reply:
Ah okay, I incorrectly assumed the WITH_ICU_I18N was all part of the same stuff.